Sail/src/Sail/entities/systems/physics/CollisionSystem.cpp

```diff
@@ -24,38 +26,144 @@ void CollisionSystem::provideOctree(Octree* octree) {
 	m_octree = octree;
 }
 
+
 void CollisionSystem::update(float dt) {
-	for (auto& e : entities) {
-		auto movement = e->getComponent<MovementComponent>();
-		auto collision = e->getComponent<CollisionComponent>();
-		auto boundingBox = e->getComponent<BoundingBoxComponent>();
-		auto csc = e->getComponent<CollisionSpheresComponent>();
+	constexpr size_t NR_OF_JOBS = 16;
+	constexpr size_t lastJob = NR_OF_JOBS - 1;
```

Comment:
if NR_OF_JOBS is capitilized, shouldnt lastJob be as well?

Reply:
lastJob has been capitalized now